Tests/PublicAPISpec.swift

```diff
@@ -119,15 +118,52 @@ class PublicAPISpec: QuickSpec {
 
             it("won't send any events until configuration has finished") {
                 RInAppMessaging.deinitializeModule()
-                reinitializeSDK()
-                messageMixerService.delay = 3.0
+                reinitializeSDK(onDependencyResolved: {
+                    messageMixerService.delay = 3.0
+                })
                 RInAppMessaging.logEvent(LoginSuccessfulEvent())
                 expect(eventMatcher.loggedEvents).toAfterTimeout(beEmpty(), timeout: 1)
                 expect(eventMatcher.loggedEvents).toEventually(haveCount(1),
                                                                timeout: .seconds(Int(messageMixerService.delay + 1)),
                                                                pollInterval: .milliseconds(500))
             }
 
+            // it will properly schedule campaigns when events are logged one after another AND
+            it("will display campaigns for matching events that were logged during config retry delay") {
```

Comment:
I can't get my head round this test 🤯

Reply:
Should I improve the description or add more context? 🙂

Reply:
Both? 😅

Reply:
Ok 😄 done. Please check

Reply:
Much more understandable 👍